### PR TITLE
fix image/NavLink

### PR DESCRIPTION
**Pull request by kuzen0211**
- add propTypes
- add alt to image
- fix navLink

There were problems with merge because I didn't know that the development branch was protected

### CODE REVIEW

File: frontend/src/pages/ThirdPage/ThirdPage.jsx [hunk 1]
```diff
@@ -1,5 +1,9 @@
+// import { useState } from 'react';
```
Comment (Aleksey2892) [inline frontend/src/pages/ThirdPage/ThirdPage.jsx:1]: You should to delete commented import

File: frontend/src/pages/ThirdPage/ThirdPage.jsx [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+import FilterComputer from '../../components/Filter/Desktop/FilterComputer/FilterComputer';
+import FilterSmartphone from '../../components/Filter/Mobile/FilterSmartphone/FilterSmartphone';
 import ProductCardList from '../../components/ProductCardList/ProductCardList';
 import styles from './ThirdPage.module.css';
+import { useLocation } from 'react-router-dom';
```
Comment (Aleksey2892) [inline frontend/src/pages/ThirdPage/ThirdPage.jsx:6]: You should to move this import on first row

**Review by Aleksey2892 — CHANGES_REQUESTED** [review]
You have to fix some things before merging

File: frontend/src/components/Filter/Desktop/FilterComputer/FilterComputer.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import styles from './FilterComputer.module.css';
+
+const FilterComputer = () => {
+  return (
+    <>
```
Comment (Aleksey2892) [inline frontend/src/components/Filter/Desktop/FilterComputer/FilterComputer.jsx:5]: there is also needed to delete empty fragment <>

File: frontend/src/components/Filter/Mobile/FilterSmartphone/FilterSmartphone.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import styles from './FilterSmartphone.module.css';
+
+const FilterSmartphone = () => {
+  return (
+    <>
+      <>
```
[the remark this answers is not in the document]
Comment (kuzen0211) [reply frontend/src/components/Filter/Mobile/FilterSmartphone/FilterSmartphone.jsx]: done

File: frontend/src/pages/ThirdPage/ThirdPage.jsx (hunk 2 above)
Comment (kuzen0211) [reply frontend/src/pages/ThirdPage/ThirdPage.jsx]: done

**Pull request merged**